ngrok.yml.example

```diff
@@ -0,0 +1,10 @@
+authtoken: YOUR_AUTH_TOKEN
```

Comment:
Although, Ngrok should not be included in the PR, IMO. It's completely distinct from the Docker component of the PR

Reply:
Good one let me remove it and add it in another PR